vault/router.go

```diff
@@ -59,6 +59,12 @@ type routeEntry struct {
 	l             sync.RWMutex
 }
 
+// loginPathsEntry is used to hold the routeEntry loginPaths
+type loginPathsEntry struct {
+	paths         *radix.Tree
+	wildcardPaths map[string]bool
```

Comment:
Add comment describing the meaning of the bool?

Reply:
The purpose of this data type is to have it behave as a set. I am happy to add a comment for clarity.